### PR TITLE
Fix part of 44: High-fi Profile Selection page

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR corresponds to UI of Profile selection page.

### Mock
https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/ad1de6e2-1d5d-4ac3-888c-58d036c7ede4/PC-MP-Profile-Chooser-

### Accessibility Scanner

- [x] Passes Accessibility Scanner test

### Screenshot
Screenshot_20191206-154901

### CODE REVIEW

File: app/src/main/res/layout/profile_chooser_fragment.xml [hunk 1]
```diff
@@ -11,24 +11,27 @@
...
     android:layout_height="match_parent"
     android:background="@color/profileChooserBackground">
+
     <TextView
       android:id="@+id/profile_select_text"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      app:layout_constraintTop_toTopOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
+      android:layout_marginStart="36dp"
```
Comment (rt4914) [inline app/src/main/res/layout/profile_chooser_fragment.xml]: Add `android:layout_marginEnd="36dp"` for RTL support

**Review by rt4914 — APPROVED** [review]
Just made a commit to fix somes issues in this. LGTM

@nikitamarysolomanpvt Please verify this as me and @veena14cs both have worked on this.

File: app/src/main/res/layout/profile_chooser_fragment.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/profile_chooser_fragment.xml]: done.

**Comment by nikitamarysolomanpvt** [comment]
Last used/visited time is missing. @veena14cs please make sure this is not missed in future.

**Review by nikitamarysolomanpvt — APPROVED** [review]
@mschanteltc iam not able to find the attributes of Administration label in the mock. PTAL .
Beside this LGTM

**Review by nikitamarysolomanpvt — CHANGES_REQUESTED** [review]
Untitled
PTAL

**Comment by veena14cs** [comment]
> Untitled
> 
> PTAL

Fixed PTAL
Screenshot_20191209-122231

**Comment by veena14cs** [comment]
> Last used/visited time is missing. @veena14cs please make sure this is not missed in future.

This needs to be fixed in low-fi issue. I have filed an issue related to this https://github.com/oppia/oppia-android/issues/531 thanks for noting it.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Comment by mschanteltc** [comment]
> @mschanteltc iam not able to find the attributes of Administration label in the mock. PTAL .
> Beside this LGTM

You can find the attributes of Administrator label on [this mock](https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/18e69208-8528-424a-a368-b2c5d530d235/PC-SP-Profile-Chooser-). Would it also be possible to include the time stamp of when each user last used the app? (Example: "Last used 4 hours ago")

**Pull request merged**